platform/src/test/kotlin/org/stellar/anchor/platform/custody/fireblocks/FireblocksPaymentServiceTest.kt

```diff
@@ -173,15 +174,15 @@ class FireblocksPaymentServiceTest {
 
   @Test
   fun `getTransactionsByTimeRange select number of items equal to limit`() {
-    FireblocksPaymentService.TRANSACTIONS_LIMIT = 2
+    fireblocksPaymentService.transactionLimit = 2
```

Comment:
Instead of mutating the object under test? Can we initialize a new `FireblocksPaymentService` with a different `transactionLimit per test?

Reply:
That would cause a lot of duplication calling the constructor though. 

We need to do: 
```
    fireblocksPaymentService = FireblocksPaymentService(fireblocksClient, fireblocksConfig, 2)
```
for every function even if we use the default `TRANSACTIONS_LIMIT`

Reply:
I think a bit of duplication in the test code is preferable to mutating the service's configuration. We should be avoiding mutation in general but especially for non-data types like `FireblocksPaymentService`.

Reply:
I created a overloaded constructor and make the `transactionLimit` read-only.